framework/src/engine/generator/schemas.ts

```diff
@@ -44,7 +44,7 @@ export const setStatusRequestSchema = {
 	$id: '/lisk/setStatusRequest',
 	title: 'Set block generation status',
 	type: 'object',
-	required: ['address', 'height', 'maxHeightGenerated', 'maxHeightPrevoted'],
+	required: ['address', 'height', 'maxHeightPreviouslyForged', 'maxHeightPrevoted'],
```

Comment:
`maxHeightGenerated` is the correct value now since it's renamed.
The commander should be adjusted instead

Reply:
Looks like I got incomplete requirements for this task 😅 

Created a new branch and PR #7650. Closing this one.